packages/react/src/VisuallyHidden/VisuallyHidden.tsx

```diff
@@ -12,17 +19,34 @@ import sx from '../sx'
  *
  * @see https://www.scottohara.me/blog/2023/03/21/visually-hidden-hack.html
  */
-export const VisuallyHidden = styled.span<SxProp>`
```

Comment:
Just wanted to share in case it was helpful (and feel free to ignore me if this is obvious 😂), when migrating styled components that are exported two helpful things I've found to double-check check are the `as` prop and `ref` support.

When we update these kinds of components these don't come along and they may be used downstream. It's hard to tell if these are breaking changes or not so I think we've been evaluating it by seeing if they are used this way in dotcom when updating.

Thankfully the integration tests tend to do a good job of double-checking this, also feel free to use Primer Query (e.g. [this link](https://primer-query.githubapp.com/?name=visuallyhidden), if you need to login then you'll have to click the link again since query params aren't persisted 😅)

Reply:
Thank you for the reminder. I'll definitely add this to my routine on components moving forward!